tests/manager/manager-ctor-ssl-deprecated-001.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): Test deprecated options
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_ssl(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$opts = array(
+    "ssl" => array(
+        "verify_peer" => false,
+        "verify_peer_name" => false,
```

Comment:
I don't think either of these two options are still used by the driver, per the context fallback table in the [`Manager::__construct()`](https://www.php.net/manual/en/mongodb-driver-manager.construct.php) documentation.

Rather, some of the options in `$deprecatedDriverOptions` below should be moved up to this array:

 * `capath`
 * `cafile`
 * `local_cert`
 * `passphrase`

Apologies if the description in [PHPC-1430](https://jira.mongodb.org/browse/PHPC-1430) didn't clarify that those were part of the context options. I likely glossed over that as they're merged into driver options by the time we start processing them in `php_phongo.c`.